N/NodeJS/NodeJS_14/build_tarballs.jl

```diff
@@ -0,0 +1,53 @@
+using BinaryBuilder
+
+name = "NodeJS_14"
+version = v"14.16.1"
+
+url_prefix = "https://nodejs.org/dist/v$version/node-v$version"
+sources = [
+    ArchiveSource("$(url_prefix)-linux-x64.tar.gz", "068400cb9f53d195444b9260fd106f7be83af62bb187932656b68166a2f87f44"; unpack_target = "x86_64-linux-gnu"),
+    ArchiveSource("$(url_prefix)-linux-arm64.tar.gz", "58cb307666ed4aa751757577a563b8a1e5d4ee73a9fac2b495e5c463682a07d1"; unpack_target = "aarch64-linux-gnu"),
+    ArchiveSource("$(url_prefix)-linux-ppc64le.tar.gz", "de6ccb9bf08520939cc2ae0507634015981604b5eb6912d031d4b7fe146f0de4"; unpack_target = "powerpc64le-linux-gnu"),
+    ArchiveSource("$(url_prefix)-darwin-x64.tar.gz", "b762b72fc149629b7e394ea9b75a093cad709a9f2f71480942945d8da0fc1218"; unpack_target = "x86_64-apple-darwin14"),
+    ArchiveSource("$(url_prefix)-win-x64.zip", "e469db37b4df74627842d809566c651042d86f0e6006688f0f5fe3532c6dfa41"; unpack_target = "x86_64-w64-mingw32"),
+    ArchiveSource("$(url_prefix)-win-x86.zip", "cfb3535a172fb792a63814deffde405466902359bedfbd884188f6fc56f97d64"; unpack_target = "i686-w64-mingw32"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/
+cp -r ${target}/*/* ${prefix}/.
+cd ${prefix}
+if [[ "${target}" == *-mingw* ]]; then
+    mkdir bin
+    chmod +x {node.exe,npm,npm.cmd,npx,npx.cmd}
+    mv {node.exe,npm,npm.cmd,npx,npx.cmd} bin
+    mv node_modules bin
+fi
+install_license LICENSE
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("x86_64", "linux"),
+    Platform("aarch64", "linux"),
+    Platform("powerpc64le", "linux"),
+    Platform("x86_64", "macos"),
+    Platform("x86_64", "windows"),
+    Platform("i686", "windows"),
+]
```

Comment:
So I would just say it would be great if we could cover the same platforms here that I have [here](https://github.com/davidanthoff/NodeJSBuilder/blob/master/build.jl#L19).

Also, I would suggest we add `MacOS(:aarch64)` and just package the x64 binaries into that artifact. They then work via rosetta, and folks on M1 can use it for now. And starting with node 16 we can then start to ship the native apple silicon binaries instead.

Reply:
> Also, I would suggest we add MacOS(:aarch64) and just package the x64 binaries into that artifact.

Why is that necessary?  If someone is using the x86-64 Julia binary with Rosetta there is no need to have a fake aarch64-apple-darwin artifact, the x86-64 would just work

Reply:
But we also want it to work for folks that are using the Apple Silicon Julia binaries :)